From 565fbb424eaee4e2ea4d0345d09be4d06d4bd704 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Tue, 18 Nov 2014 13:54:23 -0500
Subject: [redhat] kabi: Adding symbol copy_page (arch/x86/lib/copy_page_64.S)

Message-id: <8f4473cd95812a96fc52f7725366e68d7211df23.1416318767.git.poros@redhat.com>
Patchwork-id: 100144
O-Subject: [PATCH RHEL7.1 18/63] kabi: Adding symbol copy_page (arch/x86/lib/copy_page_64.S)
Bugzilla: 1070274
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1070274
    https://bugzilla.redhat.com/show_bug.cgi?id=1070274

    Description:
    ============
    Used by copy_highpage(), highmem handling.
    The bitmaps are kept in highmem.
    Symbol used in drbd

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7.0 41/80] kabi: Adding symbol copy_page (arch/x86/lib/copy_page_64.S)
 Jiri Olsa <jolsa@redhat.com> (Tue,  4 Mar 2014 16:31:41 +0100)
 Description:
 ============
 Used by copy_highpage(), highmem handling.
 The bitmaps are kept in highmem.

 Reviews:
 ========
 N/A

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/copy_page
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/copy_page

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/copy_page b/redhat/kabi/kabi-module/kabi_ppc64/copy_page
new file mode 100644
index 0000000..9c14c76
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/copy_page
@@ -0,0 +1,2 @@
+#1-
+0x33b84f74 copy_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/copy_page b/redhat/kabi/kabi-module/kabi_x86_64/copy_page
new file mode 100644
index 0000000..9c14c76
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/copy_page
@@ -0,0 +1,2 @@
+#1-
+0x33b84f74 copy_page vmlinux EXPORT_SYMBOL
-- 
1.7.1